code/modules/mob/mob_say.dm

```diff
@@ -124,6 +124,8 @@
 			mods[MODE_SING] = TRUE
 		else if(key == ";" && !mods[MODE_HEADSET] && stat == CONSCIOUS)
 			mods[MODE_HEADSET] = TRUE
+		else if(key == ";" && !mods[MODE_HEADSET] && stat != CONSCIOUS)
+			chop_to = length(key) + 1
```

Comment:
```suggestion
		else if(key == ";" && !mods[MODE_HEADSET])
			if(stat == CONSCIOUS) //necessary indentation so it gets stripped of the semicolon anyway.
				mods[MODE_HEADSET] = TRUE
```

Reply:
This is about what I meant, and on second thought is a bit more clean.